Parsers/ASimWebSession/Parsers/vimWebSessionFortinetFortiGate.yaml

```diff
@@ -0,0 +1,213 @@
+    [
+      "1", "Informational", // Debug
+      "2", "Informational", // Information
+      "3", "Informational", // Notification
+      "4", "Low", // Warning
+      "5", "Low", // Error
+      "6", "High", // Critical
+      "7", "Medium", // Alert
+      "8", "High" // Emergency
+    ];     
+    CommonSecurityLog
+    | where not(disabled)
+    | where (isnull(starttime) or TimeGenerated>=starttime) and (isnull(endtime) or TimeGenerated<=endtime)
+    | where DeviceVendor == "Fortinet" 
+      and DeviceProduct  startswith "FortiGate"
+      and (Activity      == "webfilter utm passthrough"
+          or Activity    == "webfilter utm blocked")
+    | where (array_length(url_has_any) == 0 or RequestURL has_any (remove_protocol_from_list(url_has_any)))
+    | where (array_length(httpuseragent_has_any) == 0 or AdditionalExtensions has_any(httpuseragent_has_any))
+    | extend temp_SrcMatch = has_any_ipv4_prefix(SourceIP,src_or_any)
+    | extend temp_DstMatch = has_any_ipv4_prefix(DestinationIP,ipaddr_has_any_prefix)
+    | extend ASimMatchingIpAddr = case(
+        array_length(src_or_any) == 0, "-",
+        temp_DstMatch and temp_SrcMatch, "Both",
+        temp_SrcMatch , "SrcIpAddr",
+        temp_DstMatch, "DstIpAddr",
+        "No match") 
+    | where ASimMatchingIpAddr != "No match" 
+    | project-away temp_*
+    | extend 
+        EventResultDetails = "NA"
+    | where (array_length(eventresultdetails_in) == 0 or tostring(EventResultDetails) has_any(eventresultdetails_in)) 
+    | lookup EventLookup on DeviceAction 
+    | where (eventresult == '*' or EventResult =~ eventresult)
+    | project Activity,AdditionalExtensions,DestinationIP,DestinationPort,DeviceAction,DeviceInboundInterface,DeviceOutboundInterface,DeviceProduct,DeviceVersion,LogSeverity,Protocol,ReceivedBytes,SentBytes,SourceIP,SourcePort,TimeGenerated, DeviceExternalID, Type, _ItemId, Computer, EventResult, EventResultDetails, DvcAction, RequestURL, RequestContext, DestinationHostName, SourceHostName, SourceUserName, DestinationUserName
+    | project-rename 
+      Url                     = RequestURL
+      , UrlCategory           = RequestContext
+      , DstBytes              = ReceivedBytes
+      , DstInterfaceName      = DeviceOutboundInterface
+      , DstIpAddr             = DestinationIP
+      , DstPortNumber         = DestinationPort
+      , DvcHostname           = Computer
+      , EventMessage          = Activity
+      , EventOriginalSeverity = LogSeverity
+      , EventProduct          = DeviceProduct
+      , EventProductVersion   = DeviceVersion
+      , SrcBytes              = SentBytes
+      , SrcInterfaceName      = DeviceInboundInterface
+      , SrcIpAddr             = SourceIP
+      , SrcPortNumber         = SourcePort
+      , DvcId                 = DeviceExternalID
+      , EventUid              = _ItemId
+      , DstHostname           = DestinationHostName
+      , SrcHostname           = SourceHostname
```

Comment:
this is giving error. should be
, SrcHostname           = SourceHostName

Reply:
Fixed, thanks